nautobot/utilities/templatetags/buttons.py

```diff
@@ -30,59 +30,57 @@ def clone_button(instance):
 
 
 @register.inclusion_tag("buttons/edit.html")
-def edit_button(instance, use_pk=False, key="slug"):
+def edit_button(instance):
```

Comment:
This is a breaking API change and should be avoided. Better to keep these two params for now but make them no-ops.